doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -25,15 +25,143 @@
 Traceability to Reference Model.
 
 <a name="2.3"></a>
-## 2.3 Requirements.
+## 2.3 Containerised Platform Requirements.
+
+The Reference Model (RM) defines the NFVI to consist of the physical resources, virtualised resources and a software managment system.  In the virtualised world, the NFVI Software Management System (Virtualised Infrastructure Manager -- VIM) consists of the Guest Operating System, Hypervisor and, if needed, other software such as libvirt.  And the VIM is responsible for, among others, tenant management, resources management, inventory, scheduling, and access management. Now consider, the containerisation equivalent.
+
+Containerised Infrastructure Manager (CIM): a functional block that is responsible for controlling and managing the containerised NFVI compute, storage and network resources. Note CIM supports the Life Cycle Management of conatinerised and the underlying physical resources.
+
+"Architecture" in this chapter refers to NFVI + VIM (as specified in Reference Model Chapter 3) and consists of the Physical resources, virtual and container resources, and Software Management Layer (Hypervisor, Container runtime, virtual or container Orcehstrator(s), Operating System).
```

Comment:
> Software Management Layer (Hypervisor, Container runtime, virtual or container Orcehstrator(s), Operating System).

As per comments above, this is alluding to the things in parentheses as being part of the VIM, but they are part of the NFVI Software layer.

Also the below suggests container resources are in scope, whereas the discussions in #449, I'm suggesting they're not.

> virtual and container resources

Can I suggest this is re-worded to (also includes typo correction for Orchestrator(s):

> "Architecture" in this chapter refers to NFVI + VIM (as specified in Reference Model Chapter 3) and consists of the NFVI Hardware (e.g. physical resources and NFVI Software (e.g. Hypervisor, Container Runtime, virtual or container Orchestrator(s), Operating System).

Reply:
Thanks -- change made